src/components/dashboard/dashboardv1.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from "react";
+import { axiosWithAuth } from "../../utils/axiosWithAuth.js";
+import { useHistory } from "react-router-dom";
+
+import axios from "axios";
+
+// local imports
+import "./_Dashboard.scss";
+import MovieCard from "../movies/MovieCard.js";
+// temp import to verify switch statement
+import LoadingScreen from "../layout/LoadingScreen.js";
+
+const Dashboardv1 = () => {
+  let history = useHistory();
+  const [input] = useState({ file: "" });
+  const [ratings, setRatings] = useState({});
+  const [recommendations, setRecommendations] = useState([]);
+
+  const changeHandler = e => {
+    // https://developer.mozilla.org/en-US/docs/Web/API/FormData/FormData
+    // packages up form to make it able to send over https
+    let data = new FormData();
+    data.append("movies", e.target.files[0], e.target.files[0].name);
+
+    // history.location.state.userid is just where I am holding userid for now from the Register page so I do not need to implment redux.
+    axiosWithAuth()
+      // this is insantiated when a file is added to input
+      .post(`/${history.location.state.userid}/uploading`, data, {
+        headers: {
+          "Content-Type": "multipart/form-data"
+        }
+      })
+      .then(res => {
+        console.log(res);
+        setRatings(res.data);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+    // clears out previous uploaded file
+    data = new FormData();
+  };
+
+  useEffect(() => {
+    axios
+      .get(`https://api.groa.us/api/users/1111/recommendations`)
+      .then(res => setRecommendations(res.data.recommendation_json))
+      .catch(err =>
+        console.log("Something went wrong in fetching recommendations.", err)
+      );
+  }, []);
+
+  switch (true) {
+    case !history?.location?.state?.userid:
```

Comment:
If ya don't know what these ? are for look [here to learn more about option chaining methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)